### PR TITLE
Right-to-Left support for Glitter and Config

### PR DESCRIPTION
**Pull request by puzzledsab**
#1430

Can be tested by @IONILz if it builds correctly.

### CODE REVIEW

File: sabnzbd/lang.py [hunk 1]
```diff
@@ -92,99 +92,100 @@ def list_languages():
...
+    "en": ("English", "English", 0, 0),
+    "eo": ("Esperanto", "Esperanto", 0, 0),
+    "et": ("Estonian", "Eesti", 0, 0),
+    "fo": ("Faroese", "Føroyskt", 0, 0),
+    "fj": ("Fijian", "Vosa Vakaviti", 0, 0),
+    "fi": ("Finnish", "Suomi", 0, 0),
+    "fr": ("French", "Français", 0, 0),
+    "gl": ("Galician", "Galego", 0, 0),
+    "de": ("German", "Deutsch", 0, 0),
+    "he": ("Hebrew", "עִבְרִית‎", 1255, 1),
```
Comment (Safihre) [inline sabnzbd/lang.py:126]: I'd prefer `True`/`False`.

File: sabnzbd/interface.py [hunk 2]
```diff
@@ -384,6 +384,7 @@ def index(self, **kwargs):
             info["categories"] = list_cats(True)
             info["have_rss_defined"] = bool(config.get_rss())
             info["have_watched_dir"] = bool(cfg.dirscan_dir())
+            info["rtl"] = LanguageTable.get(cfg.language(), "en")[3]
```
Comment (Safihre) [inline sabnzbd/interface.py:387]: Let's introduce a function `is_rtl(lang)` in the `sabnzbd.lang` file. Nicer :)

**Review by Safihre — CHANGES_REQUESTED** [review]
If the changes are this simple it would indeed be a good addition!

**Comment by puzzledsab** [comment]
@Safihre : Do you know why the `<!---->` remains from template tags like `<!--#end if#-->` in `include_menu.tmpl`? It doesn't happen in `main.tmpl`. Is it possible to avoid?

**Comment by thezoggy** [comment]
We used it as a tactic to safeguard from bad cheetah template leaking stuff and breaking the webpage.

**Comment by Safihre** [comment]
@puzzledsab @thezoggy I never saw that before.. strange, is that really supposed to be like that?

**Comment by Safihre** [comment]
Oh right, in the `include_` files you can just use `#` without need for `<!--` and `-->`.

**Comment by Safihre** [comment]
I don't want to go quite this far. The number of RTL users is less than 0.5% of users! 
Only basic support should be added, because it adds a maintenance burden for years to come.. That's just not worth it.
Unless there's some major things wrong, this is as far as we can go.

**Comment by puzzledsab** [comment]
@Safihre : Do you mean that any particular changes should be reversed or that I shouldn't add more?

**Comment by puzzledsab** [comment]
@Safihre : I'm not going to do more changes now. Let me know if you think anything should be reverted.

**Comment by Safihre** [comment]
I pushed some changes, shifting most of the stuff to the CSS. 
We can add some more of the needed changes there, if needed.
We can also do the config in the same way, to shift the checkboxes using CSS.
If it's really just the checkboxes, it's fine by me. If it really requires HTML changes I don't want to do it..

**Comment by puzzledsab** [comment]
According to @IONILz it's a clear improvement to add dir=rtl to the config with no other changes, so any CSS changes would be a bonus. It doesn't look so good but the text is probably easier to understand if you read Hebrew.

**Comment by Safihre** [comment]
Alright, I'll try to fix that asap :)

**Pull request merged**